### PR TITLE
Clarify well-known endpoint tw-2393

### PR DESCRIPTION
**Pull request by Karuna-Mendix**
https://mendix.atlassian.net/browse/TW-2393

### CODE REVIEW

File: content/en/docs/control-center/security/set-up-sso-byoidp.md [hunk 1]
```diff
@@ -107,8 +107,11 @@ From here you can do the following:
 When adding a configuration, you will need to provide the information described below:
 
 * **Configuration Name** – A name for the IdP setup in the SSO configuration for your own reference.
-* **OpenID Connect Endpoint URL** – The well-known endpoint at your IdP from which the Mendix Platform can retrieve the configuration metadata, including all necessary endpoints and public key location information.
+* **OpenID Connect Endpoint URL** – The endpoint at your IdP from which the Mendix Platform can retrieve the configuration metadata, including all necessary endpoints and public key location information.
```
Comment (Karuna-Mendix) [inline content/en/docs/control-center/security/set-up-sso-byoidp.md]: removed "well-known" to avoid confusion.

Comment (JaapF) [reply content/en/docs/control-center/security/set-up-sso-byoidp.md]: As per OIDC specs let's call it "Issuer URL".

The Issuer URL at your IdP. If you're using Entra ID this looks like
https://login.microsoftonline.com/550e8400-e29b-41d4-a716-446655440000/v2.0
Mendix will extend this URL with "/.well-known/openid-configuration" to retrieve the configuration metadata, including all necessary endpoints and public key location information.

Comment (JaapF) [reply content/en/docs/control-center/security/set-up-sso-byoidp.md]: I've also added an exanmple for Entra, which is most used IdP.

File: content/en/docs/appstore/modules/oidc.md [hunk 2]
```diff
@@ -276,7 +276,10 @@ In this case, the OIDC client is the app you are making.
 
 5. Add the **Client Secret**.
 6. If you have the **Automatic Configuration URL** (also known as the *well-known endpoint*), enter it and click **Import Configuration** to automatically fill the other endpoints.
-    * If you don not have an automatic configuration URL, you can fill in the other endpoints manually.
+
+    {{% alert color="info" %}}If the endpoint URL does not end with `/.well-known/openid-configuration`, append it to the endpoint URL.{{% /alert %}}
+
```
Comment (JaapF) [inline content/en/docs/appstore/modules/oidc.md:281]: Probably better to say: "Typically (and according to specifications) the URL you need to enter ends with `/.well-known/openid-configuration`."

Comment (Karuna-Mendix) [reply content/en/docs/appstore/modules/oidc.md:281]: @JaapF , Done! Please check the commit: https://github.com/mendix/docs/pull/7758/commits/3b15ad329fae9174dea37359cb088bc5245a23cf

**Review by JaapF — APPROVED** [review]

**Pull request merged**